### ISSUE TITLE
CHOMP planner segmentation fault in noetic-devel

### ISSUE THREAD

**Issue by koromkorom** [ros-planning/moveit#3154]
### Description

Whenever the CHOMP planner is called in noetic-devel, it results in a segmentation fault while handling the results. Below I also propose a solution which I found after some debugging. 

### Your environment
* ROS Distro: Noetic
* OS Version: Ubuntu 20.04
* Source build
* noetic-devel

### Steps to reproduce
Using chomp in benchmark, which is also the default planner.

### Expected behaviour
Benchmark starts planning with CHOMP.

### Actual behaviour
Segmentation fault. 

The problem is, that the benchmark expects a MotionPlanDetailedResponse and uses it the first time here: 

https://github.com/ros-planning/moveit/blob/a80dd9d6a09819315e2304763f3a2dc82b4c0185/moveit_ros/benchmarks/src/BenchmarkExecutor.cpp#L914

But the description field is not initialized yet and because it is an array the call results in an segmentation fault. 

This has been fixed in melodic devel already: 
https://github.com/ros-planning/moveit/blob/183113421a34d82f527a2e26ccca0da12cfa7b62/moveit_planners/chomp/chomp_interface/src/chomp_planning_context.cpp#L36

This fix is fine, but I would do this in the planner itself, in case its solve method is called from another interface.

Maybe like this: 
```
diff --git a/moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp b/moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp
index 6385091..48210a0 100644
--- a/moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp
+++ b/moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp
@@ -239,6 +239,7 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
 
   res.trajectory_.resize(1);
   res.trajectory_[0] = result;
+  res.description_.emplace_back("plan");
 
   ROS_DEBUG_NAMED("chomp_planner", "Bottom took %f sec to create", (ros::WallTime::now() - create_time).toSec());
   ROS_DEBUG_NAMED("chomp_planner", "Serviced planning request in %f wall-seconds",
```

If that's fine with you, I could do the PR.

**Comment by simonschmeisser** [comment]
Hi @koromkorom I just stumbled across your report, sorry for nobody replying yet! Please do open a PR and link to this issue. It's easier to discuss directly there. Looks like this might be a regression from refactoring

**Comment by rhaschke** [comment]
Thanks for reporting. As `description_` might be filled already when passed into the function, I implemented a slightly different solution: #3204.
